com.kms.cura.entity/src/com/kms/cura/entity/user/PatientUserEntity.java

```diff
@@ -21,7 +24,10 @@ public PatientUserEntity(String id, String name, String email, String password,
 	}
 
 	public String getGender() {
-		return gender;
+		if("M".equals(gender)){
+			return "Male";
+		}
+		return "Female";
```

Comment:
Please extract "M" into constant. "Male" and "Female" should be put into resources file